README.md

```diff
@@ -21,31 +19,152 @@ func sayHelloTo(name: String) {
+
+Why would you want such a function? When writing Swift code, you will often need a function that transforms a piece of data in some way, returning a _new_ value so that you can work with that value the way you want to. Maybe you want to do something other than print your friend's age, such as wish them a happy birthday. Imagine you also have the following function:
+
+```swift
+func happyBirthday(age: Int) {
+    println("Happy birthday #\(age)!")
+}
+```
+
+You want to call that function with your friend's age to wish them a happy birthday. Unfortunately, `printAge` only prints your friend's age to the console. How you can you instead wish them a happy birthday?
+
+The answer is simple: Instead of `printAge`, you can write a function, `getAge`, which returns your friend's age, and then pass that returned value to `happyBirthday`.
+
+## Defining Functions That Return Values
+
+It's easy to write a function that returns a new value, instead of printing a string. Here's an example of such a function. Add this to your playground file:
+
+```swift
+func getAge(name: String) -> Int {
+    return 29
+}
+```
+
+How is this function definition different than the ones you have written before?
+
+There are two things you should notice. First of all, there is something new between the argument list (in parentheses) and the function body (in curly braces). The new stuff is an arrow (`->`) and a _type_ (in this case, `Int`). What does this mean?
+
+The arrow `->` tells Swift that this function returns a value. Before, none of the functions you wrote returned a value, so you left this out. Since `getAge` does return a value, you have to let Swift know by typing `->` after the argument list.
+
+The return type of the function comes after `->`. It indicates what type of value the function returns, and can be any type. `getAge` returns an `Int` (the value `29`), so you can add `Int` after `->`.
+
+Remember how a constant or variable of a certain type could only hold values of that type? A `String` constant cannot hold an `Int` value, for example. Likewise, functions can only return values that match their return type. `getAge` cannot return a `String` like "oompa loompa"—it _must_ return an `Int`.
+
+The second thing you should notice is that the body of `getAge` is the line `return 29`. `return` is a _keyword_ in Swift, and indicates the value that the function returns. Every function that returns a value must return some sort of value. If you don't return a value from a function that is marked as returning one, you will get an error. Try entering this function, a bad version of `getAge` that does not return a value, in your playground file, and see what error you get:
+
+```swift
+func badGetAge(name: String) -> Int {
+    print("\(name) is 29")
+}
+```
+
+You'll see this error in your playground, which tells you that your function does not return a value, even though your promised Swift that it would:
+
```

Comment:
I think I would add a small note here stating that print is not a function which returns anything (or something like that). I mean... it can be easily inferred BUT there might be a handful of students who might think for some reason that it is returning something.